docs/includes/_resource-server-description.mdx

```diff
@@ -4,5 +4,5 @@ Use these guidelines to determine the Resource Server settings of your applicati
 | --- | --- |
 | References a Resource Server,  |  The application can include [scopes](/docs/next/api-scopes) in issued [access tokens](/docs/next/api-token-overview). The scopes your app can issue are configured in the `allowed_scopes` setting and must be equal to or a subset of the resource server's available scopes. |
 | Doesn't reference a Resource Server  |  The application can only be used to provide authentication (identity) but not authorization (access). Therefore, the scopes element in access tokens will be empty.  |
-| Doesn't need to provide multiple access levels, like admin vs. user,  |  Does not necessarily need a resource server. <mark>the introspect api will reject tokens w/o scopes though right?</mark> |
+| Doesn't need to provide multiple access levels, like admin vs. user,  |  Does not necessarily need a resource server. |
```

Comment:
> <mark>the introspect api will reject tokens w/o scopes though right?</mark>

@tomsik68, any input on this question? The context is [this](https://developer.beyondidentity.com/docs/next/add-resource-server) page.

Reply:
Thanks for the information! I think it may be sufficient to just remove this comment from here. Should we add this info, for the reader's sake, elsewhere?

Reply:
I agree removing this comment from here is the right call. I don't think this information is required in the documentation as we already have [one page for introspect endpoint](https://developer.beyondidentity.com/docs/next/validate-access-tokens#introspection-endpoint) and we also link to [the relevant RFC](https://datatracker.ietf.org/doc/html/rfc7662) from that page.

Reply:
Back when I was testing these scenarios I saw some failures when the tokens had no scopes, but I must have been mistaken. I just re-tested with both Basic and Bearer auth, and the result comes back fine, reporting the scopes accurately.